internal/testutils/olm.go

```diff
@@ -38,7 +38,7 @@ endif
 ifeq ($(IS_CHANNEL_DEFAULT), 1)
 PKG_IS_DEFAULT_CHANNEL := --default-channel
 endif
-PKG_MAN_OPTS ?= $(FROM_VERSION) $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
+PKG_MAN_OPTS ?= $(PKG_FROM_VERSION) $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
```

Comment:
It shows required an entry + migration step for who is using this pkg manifests scaffold on the projects.

Reply:
They aren't scaffolded at all, the user has to add this themselves, so I don't think this is necessary. However it doesn't hurt to add one.

Reply:
They are not scaffolded. However, the user might have followed the docs and added these steps.
Then, we should provide the guidance to migrate it in the changelog.

Reply:
I saw that besides your comment you added the steps :https://github.com/operator-framework/operator-sdk/pull/4303/files#diff-53c8ad93e1bd0a81cd5a8264eedcdedc3106a4aad6d37e51bff38c42ad7f4704R9-R13 👍 

So all good. Thank you.